python/setup.py

```diff
@@ -108,6 +108,7 @@ def get_packages(self):
     "ipaddr",
     "pycryptodomex",
     "pyopenssl",
+    "kubernetes",
```

Comment:
We have a "kubernetes" in setup_spec.extras for installing.

If there are other issues, we need to improve extras instead statically installing these optional components.